.codeclimate.yml

```diff
@@ -0,0 +1,38 @@
+engines:
+  rubocop:
+    enabled: true
+  brakeman:
+    enabled: true
+  eslint:
+    enabled: true
+  csslint:
+    enabled: true
+  coffeelint:
+    enabled: true
+    checks:
+      max_line_length:
+        enabled: false
+  duplication:
+    enabled: true
+    exclude_paths:
+      - "spec/"
+    config:
+      languages:
+        ruby:
+          mass_threshold: 30
```

Comment:
Remove this config, use defaults.
@moisesnandres add the Ruby engines in org-api (like Reek).

Reply:
ok, I will